externallib.php

```diff
@@ -127,10 +127,12 @@ public static function record_vote($discussionid, $postid, $ratingid, $sesskey)
         $ownerrating = \mod_moodleoverflow\ratings::moodleoverflow_get_reputation($moodleoverflow->id, $postownerid);
         $raterrating = \mod_moodleoverflow\ratings::moodleoverflow_get_reputation($moodleoverflow->id, $USER->id);
 
+        $canseeowner = $moodleoverflow->anonymous && $USER->id != $postownerid;
```

Comment:
In a question-only anonymous instance with several answers, the reputation of the owner is not directly updated when the non-anonymous answer has been upvoted. Only the rater reputation is increased. Only after reloading the page, the new increased reputation of the owner is displayed.

Reply:
Oh, yes, seems like I have overlooked the externallib after implementing the 'only questions are anonymous'-Setting